packages/core/src/controllers/ApiController.ts

```diff
@@ -272,17 +272,21 @@ export const ApiController = {
     await ApiController.fetchRecommendedWallets()
   },
 
-  prefetch() {
+  prefetchWalletImages() {
     const promises = [
       ApiController.fetchFeaturedWallets(),
       ApiController.fetchRecommendedWallets(),
       ApiController.fetchNetworkImages(),
```

Comment:
This needs to be called on init as well else network btton doesn't properly render. 
Maybe a cool optimizaton is loading this on the button render/init instead of here